scripts/fit_pgd_online.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import numpy as np
+import sys, os
+
+sys.path.append('../flowpm/')
+import flowpm
+import pickle
+from absl import app
+from absl import flags
+import flowpm.tfpower as tfpower
+import flowpm.scipy.interpolate as interpolate
+from flowpm.tfpower import linear_matter_power
+from flowpm import tfpm
+import jax_cosmo as jc
```

Comment:
We shouldn't be using jaxcosmo, it's not a dependency in flowpm. Instead we have nbodykit if we want.

Reply:
can we compute the halofit power spectrum by nbodykit?

Reply:
Yes, see my PR